shuup_tests/front/test_methods_phase.py

```diff
@@ -39,13 +44,33 @@ def test_method_phase_basic(rf, get_method, data, method_id):
     assert method.enabled
 
     view = MethodsPhase.as_view()
+
+    # To make method visible, basket must be available
+    person = get_person_contact(admin_user)
     request = apply_request_middleware(rf.get("/"))
+    request.shop = shop
+    request.user = admin_user
+    request.person = person
+    request.customer = person
+    basket = get_basket(request)
+
+    # add product to bakset
+    supplier = get_default_supplier()
+    product = create_product(printable_gibberish(), shop=shop, supplier=supplier, default_price=50)
+    basket.add_product(supplier=supplier, shop=shop, product=product, quantity=2)
+    assert basket.product_count, "basket has products"
+
+    basket.save()
+    request = apply_request_middleware(request, user=admin_user, person=person, customer=person, basket=basket)
```

Comment:
Your commit message:
> Methods phase now requires basket

This is a backward incompatible change, though quite edge case. So, is this OK? Can some code (other than our tests) rely on methods phase working without a basket?

Btw. you're also adding products to the basket. Does the methods phase need products in the basket too?